azure_costs_exporter/prometheus_collector.py

```diff
@@ -68,6 +68,10 @@ def _get_azure_data(self, month=None):
         rsp = requests.get(url, headers=headers, timeout=10)
         rsp.raise_for_status()
 
+        if rsp.text.startswith('"Usage Data Extract"'):
```

Comment:
I actually wanted that the exporter fails if wrong JSON is sent, so that this doesn't go unnoticed. The output used in the test is delivered every month so I wanted to correct for that without triggering an error on the Prometheus side. I could simply check if the response starts with a { ...